packages/mroonga-package-task.rb

```diff
@@ -210,10 +214,19 @@ def docker_image(os, architecture)
     "ghcr.io/mroonga/package-#{super}"
   end
 
+  def download_url(file_name)
+    "https://github.com/mroonga/mroonga/releases/download/v#{@version}/#{file_name}"
+  end
+
   def built_package_url(target_namespace, target)
-    url = "https://github.com/mroonga/mroonga/releases/download/v#{@version}/"
-    url << "#{@package}-#{target}.tar.gz"
-    url
+    file_name = ""
+    case target_namespace
+    when :source
+      file_name = "#{target}"
+    else
+      file_name = "#{@package}-#{target}.tar.gz"
+    end
+    download_url(file_name)
```

Comment:
We don't need to define `download_url` if we use `case when :source` in `built_package_url`:

```ruby
  def built_package_url(target_namespace, target)
    url = "https://github.com/mroonga/mroonga/releases/download/v#{@version}/"
    if target_namespace == :source
      url << target
    else
      url << "#{@package}-#{target}.tar.gz"
    end
    url
  end
```